### DISCUSSION TITLE
using custom props in the API (v2.0.0)

### DISCUSSION THREAD

**Discussion by JeroenHaverkort** [plausible/analytics#3866] [Self-Hosted Support]
I am testing out the API and am encountering some issues with the timeseries query regarding custom property filters. When using the dashboard you can view any metric in the main view when a custom prop filter is active (visit_duration, visits, etc.). However in the timeseries endpoint it returns an error when including the visit_duration or views_per_visit metrics. error:
![image](https://github.com/plausible/analytics/assets/93723087/0d3187f0-7967-4df4-81d4-1b6d0187539e)
Why is possible at the dashboard but not the API?

**Comment by JeroenHaverkort** [comment]
Also: why is the hourly interval not available in the API

**Reply by xavicorral** [reply]
That would be great!
